src/app/qgsattributetabledialog.h

```diff
@@ -175,6 +177,35 @@ class APP_EXPORT QgsAttributeTableDialog : public QDialog, private Ui::QgsAttrib
     void filterEdited();
     void filterQueryChanged( const QString &query );
     void filterQueryAccepted();
+
+    /**
+    * handle the expression (save or delete) when the bookmark button for stored
```

Comment:
Capitalization, full stops at end of sentences. Also we follow the Qt convention of "HandleS..." / "OpenS...